Init Commit

krislopez99 opened the pull request.

krislopez99 committed:
Init Commit

jcohen98 commented on sendEmail.py:
```diff
@@ -0,0 +1,777 @@
+				<meta charset="UTF-8">
+				<meta http-equiv="X-UA-Compatible" content="IE=edge">
+				<meta name="viewport" content="width=device-width, initial-scale=1">
+				<title>*|MC:SUBJECT|*</title>
```
This title shows up as |MC:SUBJECT| when you send the email. Can we change this?

jcohen98 commented on sendEmail.py:
```diff
@@ -0,0 +1,777 @@
+			*/
+				#templateFooter .mcnTextContent a,#templateFooter .mcnTextContent p a{
+					/*@editable*/color:#656565;
+					/*@editable*/font-weight:normal;
```
Why are there all these commented out @editable tags?

jcohen98 commented on sendEmail.py:
```diff
@@ -0,0 +1,777 @@
+	
+	s = smtplib.SMTP('smtp.gmail.com:587')
+	s.starttls()
+	s.login('info@drexelsmarthouse.com','***************')
```
Instead of filling this in through the script itself, best practice is to read from some sort of config or .properties file. There are many different ways to do this and file types you can use. If you need, I have a python class already written for this

jcohen98 commented on testScript.py:
```diff
@@ -0,0 +1,163 @@
+	
+	rootA = Tk()
+	rootA.configure(background='#24336C')
+	rootA.title('Scan Card')	
```
Is there an easy way to always keep the card access window on top of all other windows? Whenever we have another screen open, like a grafana dashboard, when a new window is made (EX: The "You signed in" window), it appears behind the open web browser

jcohen98 commented on testScript.py:
```diff
@@ -0,0 +1,163 @@
+if __name__ == "__main__":
+	firebase = firebase.FirebaseApplication('https://dsh-card-reader.firebaseio.com/', None)
+	scope = ['https://spreadsheets.google.com/feeds']
+	credentials = ServiceAccountCredentials.from_json_keyfile_name('creds.json', scope)
```
Seems like you did the proper thing with credentials down here

krislopez99 committed:
Code fix

Solved token issue, resized alert window, formatted text better, stored creds in a file for upload, used regex to ensure drexel id is used

krislopez99 commented:
All solutions have been added, a secondary file is needed for all links and passwords

jcohen98 commented on sendEmail.py:
```diff
@@ -0,0 +1,624 @@
+#!usr/bin/emv python
```
Need to change the emv to "env"

jcohen98 replied:
The full line should be #!/usr/bin/env python

jcohen98 commented on testScript.py:
```diff
@@ -0,0 +1,174 @@
+#!usr/bin/emv python
```
again change emv to env

jcohen98 replied:
The full line should be #!/usr/bin/env python

krislopez99 committed:
Changed "emv "to "env"

krislopez99 commented:
changes made, "emv" to "env"

krislopez99 committed:
added "/"
